pyproject.toml

```diff
@@ -11,7 +11,7 @@ license = {file = "LICENSE.md"}
 authors = [
     {name = "MLCommons", email = "chakra@mlcommons.org"},
 ]
-dependencies = ["protobuf==4.*", "graphviz", "networkx", "pydot"]
+dependencies = ["protobuf==5.*", "graphviz", "networkx", "pydot"]
```

Comment:
Could you please explain why updating the protobuf version was necessary?

Reply:
There were some installation conflicts with protobuf version 4. 
```
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/_pytest/assertion/rewrite.py:178: in exec_module
    exec(co, module.__dict__)
tests/jsonizer/test_jsonizer.py:5: in <module>
    from chakra.schema.protobuf.et_def_pb2 import GlobalMetadata
/opt/hostedtoolcache/Python/3.10.14/x64/lib/python3.10/site-packages/chakra/schema/protobuf/et_def_pb2.py:9: in <module>
    from google.protobuf import runtime_version as _runtime_version
E   ImportError: cannot import name 'runtime_version' from 'google.protobuf' (/opt/hostedtoolcache/Python/3.10.14/x[64](https://github.com/mlcommons/chakra/actions/runs/10691857371/job/29680367619#step:7:65)/lib/python3.10/site-packages/google/protobuf/__init__.py)
```

This can also be seen in other outstanding PRs in the repo. Changing to protobuf 5 fixed the issue.